.github/bin/retry

```diff
@@ -10,9 +10,9 @@ x() {
 max_retry_time_seconds=$(( 30 * 60 ))
```

Comment:
Commits should be order in the following order:
 - fix stuff
 - add CI job
 
 That way all commits have a chance to pass the CI.
 
 Also please extract more commits. I see that you are adding `"` frequently, please extract separate commit with only this so we could see other changes not mixed with it. It is easier to not to overlook things.